llvm/src/main/resources/org/bytedeco/llvm/include/NamedMetadataOperations.h

```diff
@@ -0,0 +1,99 @@
+/*
+ * Copyright (C) 2021 Mats Larsen
+ *
+ * Licensed either under the Apache License, Version 2.0, or (at your option)
+ * under the terms of the GNU General Public License as published by
+ * the Free Software Foundation (subject to the "Classpath" exception),
+ * either version 2, or any later version (collectively, the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *     http://www.gnu.org/licenses/
+ *     http://www.gnu.org/software/classpath/license.html
+ *
+ * or as provided in the LICENSE.txt file that accompanied this code.
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#ifndef NAMED_METADATA_OPERATIONS_H
+#define NAMED_METADATA_OPERATIONS_H
+
+#include "llvm/IR/LLVMContext.h"
+#include "llvm/IR/Metadata.h"
+#include "llvm-c/Core.h"
+#include "llvm-c/Types.h"
+
+namespace llvm {
+
+/**
+ * Exact re-implementation of LLVMGetNamedMetadataNumOperands without providing
+ * a LLVMModuleRef parameter
+ *
+ * Pass a LLVMNamedMDNodeRef instead, which is the true owner for the data
+ * anyways
+ *
+ * See /llvm/lib/IR/Core.cpp for original implementation
+ */
+extern "C" unsigned NamedMDNodeGetOperandCount(LLVMNamedMDNodeRef NodeRef) {
```

Comment:
Function names are of course, up for debate. Let me know what you think (looking at the discussion in #869). We should probably establish a good convention for these names in case we decide to try the stuff we talked about in #475

Reply:
I suppose `getNamedMDNodeNumOperands()`, `getNamedMDNodeOperand()`, and `addNamedMDNodeOperand()` would make most sense?

Reply:
Did getNamedMDNodeOperands as its in plural.